package.json

```diff
@@ -24,7 +24,7 @@
     "testTypes": "tsc --noEmit --jsx react typeTest.tsx"
   },
   "peerDependencies": {
-    "react": ">=15 || ^0.14.7"
+    "react": "^16.8.0"
```

Comment:
Adding a hook means that this is now stuck with 16.8. I made corresponding changes to the devdeps.